### PR TITLE
fix(docs): avoid error retrieving json data #11439

### PR DESCRIPTION
**Pull request by nijova**
fix issue #11439

### CODE REVIEW

**Comment by nijova** [comment]
@aishwarya24 I am using node v16.14.0

File: docs/content/latest/quick-start/build-apps/nodejs/ycql.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ async.series([
     client.execute(select, function (err, result) {
       if (err) return next(err);
       var row = result.first();
-      const city = JSON.parse(row.location).city;
+      const city = row.location.city;
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/nodejs/ycql.md:110]: This change is correct, but we also need line 97 to change, as the INSERT statement is missing `location`:

```js
const insert = "INSERT INTO ybdemo.employee (id, name, age, language, location) " +
```

**Review by polarweasel — CHANGES_REQUESTED** [review]
One further change required to get the code to actually run... If you can make it on each file, this is good to go. Thank you!

File: docs/content/latest/quick-start/build-apps/nodejs/ycql.md (hunk 1 above)
Comment (nijova) [reply docs/content/latest/quick-start/build-apps/nodejs/ycql.md:110]: @polarweasel I made the changes as suggested. But it looks to me like I messed up with git. So I'll see if I can fix that.

**Comment by nijova** [comment]
Cant figure this out, so I think it's easiest to close this and make a new pr. (please see #11545)

**Pull request closed without merging**